internal/client/conn.go

```diff
@@ -0,0 +1,305 @@
+package client
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"net"
+	"time"
+
+	"github.com/gortc/turn"
```

Comment:
if possible remove `gortc` dependency. I am just worried about not being able to fix things outside our control.

Reply:
Yes, I agree and that's the goal as we discussed.
I am thinking about taking two steps.

1. Redesign TURN client in pion/turn (this issue)
1. Remove dependencies on gortc/*

Even in the first step, I am trying to **only** depend on structures and STUN message builders/parsers defined in gortc/turn. Once the first step is complete and made it functional, I will work on step two.  (I really need to fix this TURN issue asap for my project..)

Reply:
Agree!